app/models/author.rb

```diff
@@ -1,5 +1,9 @@
 class Author < ActiveRecord::Base
   has_many :unit_templates
-  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
-  validates_attachment :avatar, content_type: { content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"] }
+
+  DEFAULT_AVATAR_URL = 'http://jaredsilver.name/headshot.jpg'
```

Comment:
TODO: change this once we have the default uploaded to the CDN. Must do before deploying.